cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/MyUserDTO.java

```diff
@@ -136,6 +136,9 @@
     @ApiModelProperty( value = "If the user has a valid payment method available", readOnly = true )
     private boolean paymentDataSaved;
 
+    @ApiModelProperty( value = "The user saved payment method labels", readOnly = true )
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "readOnly"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)